server/src/service/mod.rs

```diff
@@ -0,0 +1,4 @@
+pub mod collection;
+pub mod item;
+pub mod service;
+pub mod session;
```

Comment:
Should all of those be public? I don't think so

Reply:
`pub(crate)` perhaps?

Reply:
Done, thanks!